src/pages/Register.js

```diff
@@ -35,11 +36,24 @@ export default class Register extends Component {
 
   render() {
     const { busy, givenName, familyName, email } = this.state
-    return div({ style: { padding: '4rem' } }, [
-      div({ style: { fontSize: 60, fontWeight: 400, color: Style.colors.title } }, [
-        logo({ size: 100, style: { marginRight: 20 } }), 'SATURN'
+    return div({
+      style: {
+        flexGrow: 1,
+        padding: '5rem', marginBottom: '-2rem',
+        backgroundImage: `url(data:image/svg+xml,${encodeURIComponent(planet)})`,
```

Comment:
Since we're not using this as an icon, you might be able to simplify this by moving the file outside of the `icons` folder, which will cause webpack to load it via file-loader instead of raw-loader. Then you could just pass the file straight to `backgroundImage`

Reply:
Cool. This should also take care of scoping styles, yes?